demo/mkldnn_quant/quant_aware/PaddleCV_mkldnn_quantaware_tutorial.md

```diff
@@ -0,0 +1,217 @@
+# Image classification models fixed-point quantization tutorial
+
+## Overview
+
+Quantization is an important method for model compression and inference performance improvement. PaddlePaddle supports two quantization strategies: `post` and `aware`. In the `post` strategy a trained model is quantized. In the `aware` strategy a model is trained for quantization and then quantized. This tutorial presents the use of the `aware` training quantization strategy to quantize an image classification model and accelerate it through DNNL optimizations. On Intel (R) Cascade Lake class CPU machines, 8-bits quantization, graph optimizations and DNNL acceleration yields performance of a quantized model up to 4 times better than of an original FP32 model. Currently, quantizable operators include `conv2d`, `depthwise_conv2d`, `mul`, `matmul`; Meanwhile, in the DNNL optimization stage, we will fuse many other ops, including `batch_norm`, `relu`, `brelu`, `elementwise_add`, etc. After quantization and fuses, INT8 models performance will be greatly improved. For details about MKL-DNN optimization users can refer to [SLIM QAT for INT8 MKL-DNN](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md)
```

Comment:
Please remove the `;` after `matmul`. Also, I would suggest an update like:
"... `matmul`. DNNL optimizations consist mainly of operator fusing passes which simplify the model graph greatly, further improving the performance. For details ..."

Reply:
This information is important and still missing. Do you mean fake quantize/dequantize ops? I changed in the morning, could you refresh the page?

Reply:
I do not know why the comment below is there. I meant this comment to be only there: https://github.com/PaddlePaddle/PaddleSlim/pull/198#discussion_r411448681